website/package.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": {
     "@docusaurus/core": "3.5.2",
     "@docusaurus/preset-classic": "3.5.2",
+    "@iconify/react": "^5.0.2",
```

Comment:
For the front-end UI we're using https://lucide.dev/icons/. Should we align?

Reply:
Lucide doesn't have the tool icons I need, for example, for Kubernetes.